app/assets/javascripts/components/student/student_conference.es6.jsx

```diff
@@ -32,21 +32,17 @@ class StudentConference extends Component {
   // --------------------------------------------------
   // Render
   // --------------------------------------------------
+  // TODO: Add Student status and Rooming information.
   render() {
-    var studentConference = this.props.student.student_conference;
+    var studentConference = this.props.student.group.conference;
     return (
       <div style={this.styles.container}>
         <h4>{'Status'}</h4>
-        <h6>{studentConference.status}</h6>
         <br />
         <h4>{'Group'}</h4>
-        <h6>{studentConference.group_letter}</h6>
-        <h6>{studentConference.team_leader_name}</h6>
-        <h6>{studentConference.junior_crew_name}</h6>
+        <h6>{studentConference.name}</h6>
```

Comment:
I'm assuming you'll be rendering the other data later through `this.props.student.group`?
Might be better to do `var group = this.props.student.group` then.